src/main/java/org/broadinstitute/consent/http/util/JsonSchemaUtil.java

```diff
@@ -45,4 +47,19 @@ public boolean isValidSchema_v1(String datasetRegistrationInstance) {
       return false;
     }
   }
+
+  public DatasetRegistrationSchemaV1 deserializeDatasetRegistration(String datasetRegistrationInstance) {
+    try {
+      String schemaString = IOUtils.resourceToString("/dataset-registration-schema_v1.json", Charset.defaultCharset());
```

Comment:
would it make sense / be possible to store this in memory so it doesn't have to reload every time someone makes a dataset?

Reply:
Excellent idea ... will do 👍🏽